template/conftest.py

```diff
@@ -136,11 +136,11 @@ def navigator(backend, firmware, golden_run):
         raise ValueError(f"Device '{firmware.device}' is unsupported.")
 
 
-@pytest.fixture(autouse=True)
-def use_only_on_backend(request, backend):
+@pytest.fixture
+def use_only_on_backend(request, backend_name):
```

Comment:
Is this one really used?

Reply:
In `one` test in ragger, nowhere else, that's why we only see the issue now.

That being said this fixture makes sense so I suggest keeping it